engines/config-query-sparql/config/rdf-resolve-quad-pattern/actors/string-source.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Could you run a formatter over this, so that a consistent two-space indentation is used?